cluster/config-defaults.yaml

```diff
@@ -140,8 +141,15 @@ tokeninfo_url: "" # can be set when local tokeninfo is disabled
 skipper_local_tokeninfo: "bridge"
 {{end}}
 
+# oauth2 UI login - grant flow
+skipper_oauth2_ui_login: "true"
+skipper_oauth2_secrets_dir: /etc/skipper/oauth
```

Comment:
Do these things really need to be configurable? Why not just hardcode it depending on `skipper_oauth2_ui_login`?

Reply:
done